package.json

```diff
@@ -62,8 +62,7 @@
     "highlight.js": "^10.4.0",
     "jdenticon": "^2.2.0",
     "jsonschema": "^1.4.0",
-    "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1",
+    "lit": "^2.0.0-rc.2",
```

Comment:
:scream:  seeing this RC dependency makes me kind of nervous :smile:

Reply:
same for me